src/main/java/org/example/programmer.java

```diff
@@ -0,0 +1,8 @@
+package org.example;
+
+public class programmer implements Calculator {
```

Comment:
Class names should start with Uppercase letters as per convention. And ProgrammerCalculator would be better. Same applies for scientific.